test/parse-outputs/test_parse_outputs.py

```diff
@@ -21,42 +23,49 @@ def setup_class(cls):
+            out_file = OUTDIR / f"{algo}-pathway.txt"
 
             runner.parse_output(algo, test_file, out_file)
-            assert filecmp.cmp(OUTDIR + f"{algo}-pathway.txt", EXPDIR + f"{algo}-pathway-expected.txt", shallow=False)
+            assert filecmp.cmp(OUTDIR / f"{algo}-pathway.txt", EXPDIR / f"{algo}-pathway-expected.txt", shallow=False)
 
     def test_empty_file(self):
         for algo in algorithms:
-            test_file = INDIR + f"empty-raw-pathway.txt"
-            out_file = OUTDIR + f"{algo}-empty-pathway.txt"
+            test_file = INDIR / f"empty-raw-pathway.txt"
+            out_file = OUTDIR / f"{algo}-empty-pathway.txt"
 
             runner.parse_output(algo, test_file, out_file)
-            assert filecmp.cmp(OUTDIR + f"{algo}-empty-pathway.txt", EXPDIR + f"empty-pathway-expected.txt", shallow=False)
+            assert filecmp.cmp(OUTDIR / f"{algo}-empty-pathway.txt", EXPDIR / f"empty-pathway-expected.txt", shallow=False)
 
     def test_oi2_miss_insolution(self):
-        test_file = OI2_EDGE_CASES_INDIR + f"omicsintegrator2-miss-insolution-raw-pathway.txt"
-        out_file = OUTDIR + f"omicsintegrator2-miss-insolution-pathway.txt"
+        test_file = OI2_EDGE_CASES_INDIR / f"omicsintegrator2-miss-insolution-raw-pathway.txt"
+        out_file = OUTDIR / f"omicsintegrator2-miss-insolution-pathway.txt"
 
         runner.parse_output('omicsintegrator2', test_file, out_file)
-        assert filecmp.cmp(out_file, EXPDIR + f"empty-pathway-expected.txt", shallow=False)
+        assert filecmp.cmp(out_file, EXPDIR / f"empty-pathway-expected.txt", shallow=False)
 
     def test_oi2_wrong_order(self):
-        test_file = OI2_EDGE_CASES_INDIR + f"omicsintegrator2-wrong-order-raw-pathway.txt"
-        out_file = OUTDIR + f"omicsintegrator2-wrong-order-pathway.txt"
+        test_file = OI2_EDGE_CASES_INDIR / f"omicsintegrator2-wrong-order-raw-pathway.txt"
+        out_file = OUTDIR / f"omicsintegrator2-wrong-order-pathway.txt"
 
         runner.parse_output('omicsintegrator2', test_file, out_file)
-        assert filecmp.cmp(out_file, EXPDIR + f"omicsintegrator2-pathway-expected.txt", shallow=False)
+        assert filecmp.cmp(out_file, EXPDIR / f"omicsintegrator2-pathway-expected.txt", shallow=False)
 
     def test_duplicate_edges(self):
         for algo in algorithms:
-            test_file = DUPLICATE_EDGES_DIR + f"{algo}-raw-pathway.txt"
-            out_file = OUTDIR + f"{algo}-duplicate-pathway.txt"
+            test_file = DUPLICATE_EDGES_DIR / f"{algo}-raw-pathway.txt"
+            out_file = OUTDIR / f"{algo}-duplicate-pathway.txt"
 
             runner.parse_output(algo, test_file, out_file)
-            assert filecmp.cmp(OUTDIR + f"{algo}-duplicate-pathway.txt", EXPDIR + f"{algo}-pathway-expected.txt", shallow=False)
+            assert filecmp.cmp(OUTDIR / f"{algo}-duplicate-pathway.txt", EXPDIR / f"{algo}-pathway-expected.txt", shallow=False)
+
+    def test_meo_present_headers(self):
```

Comment:
Instead update the test_empty_file test case with MEO + other algorithms empty file formats.